bhyve-api/src/structs.rs

```diff
@@ -271,6 +271,23 @@ pub struct vm_capability {
     pub allcpus: c_int,
 }
 
+#[repr(C)]
+#[derive(Copy, Clone, Default)]
+pub struct vm_suspend {
+    /// Acceptable values defined by `vm_suspend_how`
+    pub how: u32,
+}
+
+// bit definitions for `vm_reinit.flags`
+pub const VM_REINIT_F_FORCE_SUSPEND: u64 = 1 << 0;
```

Comment:
Not to pick on this addition, but we should try to make more use of bitstruct/bitflags in propolis in general.

Reply:
Gotta start somewhere.  Done